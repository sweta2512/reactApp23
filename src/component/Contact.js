import React from 'react'

const Contact = () => {
    return (
        <div><h1 className='my-5 font-bold text-center'>Contact</h1>
            <div className=' w-6/12 m-auto flex-2 flex-column border '>
                <div className='flex'>
                    <label>Name</label>
                    <input className='border' type='text' />
                </div>
                <div className=' my-5 w-200 '>
                    <label>Email</label>
                    <input className='border border-black p-2' type='text' />
                </div>
                <div className='my-5 flex w-100'>
                    <label>Query</label>
                    <textarea className='border border-black p-2' defaultValue='this is text area'/> 
                </div>
            </div>
        </div>
    )
}

export default Contact