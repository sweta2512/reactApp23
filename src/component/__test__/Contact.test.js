import { render ,screen} from "@testing-library/react"
import Contact from "../Contact"
import "@testing-library/jest-dom";

test('Contact component should render',()=>{
    render(<Contact/>)
    let heading = screen.getByRole('heading');
    expect(heading).toBeInTheDocument();

})