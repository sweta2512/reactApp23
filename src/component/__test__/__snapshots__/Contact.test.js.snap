// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test cases for contact us page snapShot testing 1`] = `
{
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "logTestingPlaygroundURL": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
}
`;

exports[`Test cases for contact us page snapShot testing second way 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <h1
          class="my-5 font-bold text-center"
        >
          Contact
        </h1>
        <div
          class=" w-6/12 m-auto flex-2 flex-column border "
        >
          <div
            class="m-20px"
          >
            <div
              class="flex"
            >
              <label>
                Name
              </label>
              <input
                class="border"
                type="text"
              />
            </div>
            <div
              class=" my-10 w-200 "
            >
              <label>
                Email
              </label>
              <input
                class="border border-black p-2"
                type="text"
              />
            </div>
            <div
              class="my-5 flex w-100"
            >
              <label>
                Query
              </label>
              <textarea
                class="border border-black p-2"
              >
                this is text area
              </textarea>
            </div>
            <button>
              Submit
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <h1
        class="my-5 font-bold text-center"
      >
        Contact
      </h1>
      <div
        class=" w-6/12 m-auto flex-2 flex-column border "
      >
        <div
          class="m-20px"
        >
          <div
            class="flex"
          >
            <label>
              Name
            </label>
            <input
              class="border"
              type="text"
            />
          </div>
          <div
            class=" my-10 w-200 "
          >
            <label>
              Email
            </label>
            <input
              class="border border-black p-2"
              type="text"
            />
          </div>
          <div
            class="my-5 flex w-100"
          >
            <label>
              Query
            </label>
            <textarea
              class="border border-black p-2"
            >
              this is text area
            </textarea>
          </div>
          <button>
            Submit
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
